frontend/sozisel-app/src/components/ParticipantActiveSession/Modules/QuizEvent/ParticipantQuizQuestion/ParticipantQuizQuestion.tsx

```diff
@@ -0,0 +1,65 @@
+import "./ParticipantQuizQuestion.scss";
+import { Answer, ParticipantQuizQuestion } from "../../../../../graphql";
+import { Button, List, Typography } from "@material-ui/core";
+
+import { ReactElement, useEffect, useState } from "react";
+import { useParticipantQuizContext } from "../../../../../contexts/ParticipantQuiz/ParticipantQuizContext";
+
+export default function ParticipantQuizQuestion(): ReactElement {
+  const [
+    { currentQuestion, currentAnswer },
+    dispatch,
+  ] = useParticipantQuizContext();
+  const [timer, setTimer] = useState<number>(0.0);
+
+  useEffect(() => {
+    setTimer(0.0);
+  }, [currentQuestion]);
+
+  useEffect(() => {
+    let mounted = true;
+    setTimeout(() => {
+      if (mounted) {
+        setTimer(timer + 0.1);
+      }
+    }, 100);
+    return () => {
+      mounted = false;
+    };
+  }, [timer]);
+
+  const quizAnswer = (answer: Answer) => {
```

Comment:
useCallback, you are updating this component very often due to the setTimer

Reply:
When I look at this again I have a question: what is the purpose of this component? Seems like a huge hack...